web/src/app/modules/sugarloaf/components/smart/uploader/uploader.component.ts

```diff
@@ -24,7 +24,7 @@ export class UploaderComponent implements OnInit, OnDestroy {
       this.showModal = true;
     });
     this.websocketService.registerHandler('event.octant.dev/refresh', () => {
-      setTimeout(window.location.reload.bind(window.location), 1000);
```

Comment:
This setTimeout is still needed because it is possible for the page to reload before the loading API goes away.